apps/prairielearn/src/ee/auth/lti13/lti13Auth.ts

```diff
@@ -149,9 +158,58 @@ const OIDCAuthResponseSchema = z.object({
 router.post(
   '/callback',
   asyncHandler(async (req, res) => {
+    debug('callback');
+    debug(req.method);
+    debug(req.body);
+
+    // https://www.imsglobal.org/spec/security/v1p0/#step-3-authentication-response
+    OIDCAuthResponseSchema.passthrough().parse(req.body);
```

Comment:
I'd expect to see the return value of this assigned to something and used below.